neo.UnitTests/protocol.json

```diff
@@ -1,5 +1,5 @@
 {
   "ProtocolConfiguration": {
-    "SecondsPerBlock": 2
+    "MillisecondsPerBlock": 2
```

Comment:
So fast :P

Reply:
kkkkkk
2000

Reply:
updated, @shargon, can you take a look in the rest? maybe some formula or shift operators need to be adjusted.